### PR TITLE
Add UUIDPrimaryKey scope doc

### PR DESCRIPTION
**Pull request by girardinsamuel**

### CODE REVIEW

**Comment by girardinsamuel** [comment]
@josephmancuso Can I merge this into docs v3.0 ?

File: masonite-orm/models.md [hunk 1]
```diff
@@ -294,6 +294,41 @@ You can disable this behavior as well:
 User.with_trashed().all() #== SELECT * FROM `users`
 ```
 
+### Changing Primary Key to use UUID
+
+Masonite ORM also comes with another global scope to enable using UUID as primary keys for your models.
+
+Simple inherit the `UUIDPrimaryKey` scope:
```
Comment (josephmancuso) [inline masonite-orm/models.md]: Simply. Typo

File: masonite-orm/models.md [hunk 2]
```diff
@@ -294,6 +294,41 @@ You can disable this behavior as well:
...
+
+```python
+from masoniteorm.scopes import UUIDPrimaryKeyMixin
+
+class User(Model, UUIDPrimaryKeyMixin):
+  # ..
+```
+
+You should also define a UUID column with primary constraint in a migration
```
Comment (josephmancuso) [inline masonite-orm/models.md:310]: You say primary constraint here but I don't see the primary constraint?

**Review by josephmancuso — CHANGES_REQUESTED** [review]

**Comment by josephmancuso** [comment]
Sure. Just read it over and commented on a few things. After you think it's good feel free to merge it

File: masonite-orm/models.md (hunk 2 above)
Comment (girardinsamuel) [reply masonite-orm/models.md:310]: I had a question about this, cool that you're pointing it out. Is a primary key not automatically assigned to the field named "id" ?
Also can we currently do this: `table.uuid("id").primary()` ? or should we do : 
```
table.uuid("id")
table.primary("id")
```

Comment (josephmancuso) [reply masonite-orm/models.md:310]: No we can't assume that id is the primary column. I typically don't use id as primary columns actually.  For work we typically do user_id or order_id as the primary auto incrementing columns

Comment (josephmancuso) [reply masonite-orm/models.md:310]: We don't currently support the first way of setting a primary key but we would be able to add it in like 30 seconds. We currently do that with other fields like unique constraints

Comment (girardinsamuel) [reply masonite-orm/models.md:310]: Would be cleaner to be able to add it this way

Comment (josephmancuso) [reply masonite-orm/models.md:310]: I agree. Can open a ticket. Would just have to copy the way the unique method works in the blueprint class

Comment (girardinsamuel) [reply masonite-orm/models.md:310]: I'll try to do it in a small PR and then merge the doc PR

Comment (girardinsamuel) [reply masonite-orm/models.md:310]: Done here https://github.com/MasoniteFramework/orm/pull/208

**Pull request merged**